draft-ietf-quic-http.md

```diff
@@ -1007,11 +1030,16 @@ HTTP/3 uses a priority scheme similar to that described in {{!RFC7540}}, Section
 another request, which expresses the preference that the latter stream (the
 "parent" request) be allocated resources before the former stream (the
 "dependent" request). Taken together, the dependencies across all requests in a
-connection form a dependency tree. The structure of the dependency tree changes
-as PRIORITY frames add, remove, or change the dependency links between requests.
+connection form a dependency tree.
 
-The PRIORITY frame {{frame-priority}} identifies a prioritized element. The
-elements which can be prioritized are:
+When a client request is first sent or a placeholder first allocated, the
+element is dependent on the root of the priority tree. Pushed streams are
+initially dependent on the client request on which the PUSH_PROMISE frame was
+sent. In all cases, elements are assigned an initial weight of 16.
```

Comment:
This is a nice side-effect.

Reply:
Moved to #2074 so I could do more priority-related changes without blocking on this, but I agree.  :-)